projects/demo/src/pages/stackblitz/stackblitz.service.ts

```diff
@@ -0,0 +1,64 @@
+import {Injectable} from '@angular/core';
+import stackblitz, {OpenOptions, Project} from '@stackblitz/sdk';
+import {
+    TuiCodeEditor,
+    tuiRawLoad,
+    tuiTryParseMarkdownCodeBlock,
+} from '@taiga-ui/addon-doc';
+
+@Injectable()
+export class StackblitzService implements TuiCodeEditor {
+    private readonly baseProjectConfigs: Pick<Project, 'dependencies' | 'template'> = {
+        template: `typescript`,
+        dependencies: {
+            '@maskito/core': '*',
```

Comment:
why not just use the fixed version from master?

<img width="576" alt="image" src="https://user-images.githubusercontent.com/12021443/222096175-5044c74a-e12d-4997-8337-7dd74e10be20.png">

Reply:
We don't have many parrallel versions of maskito (so, we can take the latest one in stackblitz)

Reply:
You don't have `MASKITO_VERSION` for runtime?

```ts
import { MASKITO_VERSION } from "@maskito/core";
```

Reply:
aaa, okey